src/actions/index.js

```diff
@@ -55,10 +57,10 @@ export function saveCurrentProject(state) {
   return false;
 }
 
-function validateSource(language, source, enabledLibraries) {
+function validateSource(language, source, analyzer) {
```

Comment:
Okay, I renamed usage to projectAttributes while keeping the class name the same, which I think is what you meant.